modules/influxdb/build.gradle

```
@@ -3,6 +3,8 @@ description = "Testcontainers :: InfluxDB"
 dependencies {
     api project(':testcontainers')
 
-    compileOnly 'org.influxdb:influxdb-java:2.21'
-    testImplementation 'org.influxdb:influxdb-java:2.21'
+    implementation "org.influxdb:influxdb-java:2.21"
+    testImplementation "org.influxdb:influxdb-java:2.21"
```

Let's revert this change completely
```suggestion
    compileOnly 'org.influxdb:influxdb-java:2.21'
    testImplementation 'org.influxdb:influxdb-java:2.21'
```